bom/application/pom.xml

```diff
@@ -433,26 +432,24 @@
                 <version>${opensearch-testcontainers.version}</version>
             </dependency>
             <!-- OpenTelemetry components, imported as a BOM -->
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom</artifactId>
-                <version>${opentelemetry.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom-alpha</artifactId>
-                <version>${opentelemetry.version}-alpha</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
+            <!-- opentelemetry-instrumentation-bom-alpha includes the main and alpha-bom inside.-->
+            <!-- Also includes semantic conventions-->
             <dependency>
                 <groupId>io.opentelemetry.instrumentation</groupId>
                 <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
-                <version>${opentelemetry-alpha.version}</version>
+                <version>${opentelemetry-instrumentation.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.opentelemetry.javaagent</groupId>
+                        <artifactId>*</artifactId> <!-- Exclude all artifacts in this group -->
+                    </exclusion>
+                    <exclusion>
+                        <groupId>io.opentelemetry.javaagent.instrumentation</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Note that if this drops the constraints from the Quarkus tree itself, I'm for keeping them so that the generated BOM (see Alexey's PR) and the constraints in the project are consistent.

Reply:
I tested removing a group id that we really need and nothing happens.
Exclusions on bom artifacts don't seem to work, or at least it does something that is not obvious. 
We can keep it there, just in case, but will add a comment.

Reply:
Let's remove these exclusions

Reply:
ok